src/components/DocsHome/index.tsx

```diff
@@ -475,7 +475,18 @@ const DocsHomePage: FC = () => {
                 className="button-white button-with-icon"
                 href="/docs/current/developer-docs/getting-started/overview-of-icp"
               >
-                Start building
+              Learn more
```

Comment:
if you nest the `<p>` inside the `<div>` up top, the two buttons should end up on the same axis instead of broken onto two lines

```
<div className={"flex flex-row gap-2 flex-wrap">
  <p className="mb-0">
    <Link
      className="button-white button-with-icon"
      href="/docs/current/developer-docs/getting-started/overview-of-icp"
    >
      Start building
      <LinkArrowRight />
    </Link>
  </p>
  <p className="mb-0">
    <Link
      className="button-white button-with-icon"
      href="https://github.com/dfinity/examples/"
    >
      Start coding
      <LinkArrowRight />
    </Link>
  </p>
</div>
```

Reply:
Fixed 👍🏻